README.rst

```diff
@@ -90,6 +90,12 @@ Get in touch
 .. _on GitHub: https://github.com/sphinx-doc/sphinx
 .. _mailing list: https://groups.google.com/forum/#!forum/sphinx-users
 
+Please adhere to our `code of conduct`__.
```

Comment:
But how will user/contributor concretely adhere?

Reply:
How about adding a hyperlink to issue template?

Reply:
Makes sense. I think you have the necessary permissions on GitHub (I don't have them).

Reply:
No, issue templates are managed under `.github/ISSUE_TEMPLATES/`. So you can modify them all :-)